### PR TITLE
Add Search filter

### PR DESCRIPTION
**Pull request by bmclaughlin**
add-search_actions

add-search_rules

### CODE REVIEW

**Comment by AllenBW** [comment]
@ncameronbritt  they are not

also another thought @bmclaughlin might want to checkout https://github.com/RedHatInsights/insights-frontend-components/blob/master/src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

**Comment by bmclaughlin** [comment]
@AllenBW 800 it is.

File: src/SmartComponents/Filters/Filters.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    );
+
+    render () {
+        const { rules } = this.props;
+        const { text } = this.state;
+
+        return (
+            <Grid className='advisorFilters' gutter={ 'md' }>
+                <GridItem span={ 4 }>
+                    <TextInput value={ text } onChange={ this.changeFilterValue } aria-label='Search' />
```
[the remark this answers is not in the document]
Comment (bmclaughlin) [reply src/SmartComponents/Filters/Filters.js]: Done, good catch.

**Comment by AllenBW** [comment]
Last thing, throwing this on the rules page, then I think its good for a merge!

**Comment by bmclaughlin** [comment]
@AllenBW, added to Rules page.

**Comment by AllenBW** [comment]
Looks like there's still some work to be done to get this looking like the mock, @bmclaughlin can yah put the search input on the same line as the checkbox and results as per the mockup? 
### What it looks like on the rules page presently
screen shot 2019-02-01 at 4 03 35 pm

**Comment by AllenBW** [comment]
screen shot 2019-02-04 at 10 46 03 am
@bmclaughlin just pushed up a commit that does this... ☝️ 

but theres a few other things this work could use... the loading component on the results text is kinda odd looking, a card that appears that disappears, might want to just do text there...

second thing, a little trickier maybe, integrating the  checkbox with the search... that probably should be done... if the filters component had an option to add additional external filters...

**Pull request closed without merging**